io.brackets.Brackets.json

```diff
@@ -0,0 +1,71 @@
+{
+  "app-id": "io.brackets.Brackets",
+  "branch": "stable",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.freedesktop.Sdk",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "brackets",
+  "rename-desktop-file": "brackets.desktop",
+  "rename-icon": "brackets",
+  "separate-locales": false,
+  "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=pulseaudio",
+      "--share=network",
+      "--device=dri",
+      "--filesystem=host",
+      "--talk-name=org.freedesktop.Notifications",
+      "--talk-name=org.gtk.vfs.*",
+      "--talk-name=org.gnome.GConf",
+      "--filesystem=host"
+  ],
+  
+  "modules": [
+    {
+      "name": "brackets",
+      "buildsystem": "simple",
+      "build-commands": [
+        "if [[ $(uname -m) = 'x86_64' ]]; then ar x Brackets.Release.release-1.13-prerelease-3.64-bit.deb; else ar x Brackets.Release.release-1.13-prerelease-3.32-bit.deb; fi",
```

Comment:
Can be simplified to `ar x Brackets.Release.release-1.13-prerelease-3.*.deb`.

Reply:
Yep, but full paths always reduce the risks.